composer.json

```diff
@@ -66,6 +66,7 @@
         "prestashop/gridhtml": "^2",
         "prestashop/gsitemap": "^4",
         "prestashop/pagesnotfound": "^2",
+        "prestashop/phpstan-prestashop": "^2.0",
```

Comment:
This should go to the `require-dev` part

Reply:
Oops :smile: